assets/css/general.css

```diff
@@ -396,23 +395,6 @@ li.language:hover {
     max-height: 250px;
 }
 
-.has-error .control-label,
-.has-error label {
-    color: #dc3545;
-}
-
-.has-error .form-control {
-    border-color: #dc3545;
-}
-
-a {
-    color: #17a2b8;
-}
-
-a:hover {
-    color: #0a6977;
-}
-
```

Comment:
@janinavelasco9 why did you remove that ? Now in the appointment process, when the "customer informations" form is not valid, the fields are no more marked as red.

Reply:
i must have removed this for something inside the master branch. control-label isnt a thing anymore with bootstrap4. i will add back though and look into it.